gaffer-as-a-service/gaas-jwt-sidecar/src/test/java/uk/gov/gchq/gaffer/sidecar/controller/SidecarControllerTest.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.gchq.gaffer.sidecar.controller;
```

Comment:
Tests still require license, this should have failed checkstyle

Reply:
Fixed